torch/distributed/rpc/__init__.py

```diff
@@ -164,7 +164,7 @@ def init_rpc(
         # keep it alive.
         global rendezvous_iterator
         rendezvous_iterator = dist.rendezvous(
-            rpc_backend_options.init_method, rank=rank, world_size=world_size
+            rpc_backend_options.init_method, rank=rank, world_size=world_size, init_rpc=True
```

Comment:
curious, is `world_size=-1` sufficient to identify that this is a dynamic group? I would assume `rendezvous` / `store` does not need to now whether this is RPC or not?